tests/integration/conftest.py

```diff
@@ -216,10 +216,10 @@ def linode_with_label():
         .rstrip()
     )
 
-    yield result
-
     res_arr = result.split(",")
     linode_id = res_arr[4]
+    yield result, linode_id
+
```

Comment:
Might be cleaner to make a new fixture within the test file since fixtures in conftest are meant to be generic. This yield might be too specific to be used by other test suites

Reply:
i have altered the other files where respective fixture is used that way we can reuse the same method any how i will create on more in the conftest with linode create since it would generic in may test if we needs to optimize